quarkus-debezium-parent/quarkus-debezium-postgres-parent/runtime/src/main/java/io/quarkus/debezium/configuration/DebeziumEngineConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.quarkus.debezium.configuration;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+
+/**
+ * Debezium configuration.
+ */
+@ConfigMapping(prefix = "quarkus.debezium")
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public interface DebeziumEngineConfiguration {
+    /**
+     * Configuration properties
+     */
+    Map<String, String> configuration();
```

Comment:
I'm wondering if we should just use quarkus.debezium.* instead of quarkus.debezium.configuration.*.

Reply:
I think it's feasible

Reply:
I absolutely agree @mfvitale, I really would prefer to avoid unnecessary config option name bloat.